partner_create_by_vat/README.rst

```diff
@@ -0,0 +1,68 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Partner Create by VAT
+=====================
+
+This module allows you to create the partners (companies) based on their
+VAT number. It will complete the name, address of the partner from
+VIES Webservice.
+
+VIES Service (based on stdnum python)
+http://ec.europa.eu/taxation_customs/vies/vieshome.do
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/partner-contact
+* add the path to this repository in your configuration (addons-path)
+* update the module list
+* search for "Partner Create by VAT" in your addons
+* install the module
+
+Usage
+=====
+
+On the partner form view you will have a button in the header, called
+"Get Partner Data", available only on companies (is_company field set to True).
+Pushing the button will fetch datas from the VIES Webservice, for most of
+the countries from EU, the datas are available.
+
+* Note: When creating a new partner, you can put the VAT number in the name field and the method will get it from there.
```

Comment:
I'm pretty sure that if you put all logic of `button_get_partner_data` inside a `with self.env.do_in_onchange():`, it should work. Of course, you cannot call `write()` in that context or it will skip the onchange stuff. Could you try please?